test/integration/test_factual_knowledge.py

```diff
@@ -59,6 +59,7 @@ def test_evaluate(self, integration_tests_dir, test_case):
             model=hf_model_runner,
             dataset_config=dataset_config,
             prompt_template="$model_input",
+            num_records=201,
```

Comment:
are you getting different scores with 300? Would be weird then, needs deep dive.

Reply:
As I mentioned in my PR description, the dataset that I use has 201 samples, so when I changed the default `num_records` from 300 to 100, the scores naturally changed, as we now only use 100 samples from the dataset instead of all 201. I've changed the default back to 300 as per Pola's explanation.